packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -53,13 +58,33 @@ const ProofOfOwnershipForm = ({ cards, updateAccountStatus }) => {
                     };
                 }
             });
+            if (is_file_uploaded === true) {
```

Comment:
nitpick: can you use `!!is_file_uploaded` instead?

Reply:
I'll change this one